envboard.yaml

```diff
@@ -0,0 +1,30 @@
+# Install the conda environment by running the following command in your conda base environment:
+# conda env create -f envboard.yaml
+name: 'envboard'
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - ipykernel
+  - matplotlib>=3.2.2
+  - scikit-learn>=1.1.3
+  - requests>=2.24.0
+  - graphviz
+  - python-graphviz
+  - eli5
+  - shap
+  - jinja2
+  - altair_saver
+  - selenium<4.3.0
+  - pandas<1.5
+  - imbalanced-learn
+  - pip
+  - lightgbm
+  - docopt-ng==0.8.1
+  - requests 2.27.1
+  - numpy==1.23.4
+  - pandas==1.4.4
+  - altair==4.2.0
+  - altair_saver
+  - scikit-learn==1.1.3
+    
```

Comment:
I think we're also missing vl-convert-python here. We need that package to run the code Joel provided to save the images.